src/main/resources/application.yml

```
@@ -0,0 +1,77 @@
+  jpa:
+    hibernate:
+      ddl-auto: update
+    show-sql: true
+    properties:
+      hibernate:
+        dialect: org.hibernate.dialect.PostgreSQLDialect
+        format_sql: true
+
+# Flyway configuration
+  flyway:
+    recreate-db: true 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)